spec/factories/resturant_factory.rb

```diff
@@ -0,0 +1,11 @@
+
+FactoryGirl.define do
+  factory :restaurant do
+    id { '1' }
```

Comment:
why did you use a constant id here? also your `restaurant_id` is defined as `integer`